### ISSUE TITLE
Disable JS for remote content

### ISSUE THREAD

**Issue by EnriqueDMeneses** [westmonroe/pdf-puppeteer#25]
It would be great to be able to pass in args for the page object. More specifically `page.setJavaScriptEnabled(enabled)`

This is handy when remoteContent is true but you don't want JS to run.

**Comment by cr7boulos** [comment]
@EnriqueDMeneses 

Yes, given there is another request for additional access to the puppeteer API from pdf-puppeteer, a refactor of our current API might be in order to make it easier to incrementally add functionality that does not introduce breaking changes or generate an unwieldy number of function parameters. I'll mark this as an enhancement in the meantime.
